### PR TITLE
Heroes loadout patch

### PR DESCRIPTION
**Pull request by HeroesMaker**
created new faction, change some stuff in riflemanCCO and afranian army structure, update faction list to include new loadout. ( first time doing this shit )

### CODE REVIEW

File: loadouts/faction-list.sqf [hunk 1]
```diff
@@ -132,6 +132,10 @@ switch (toUpper(_faction)) do {
     case "FRWINTER": {
         #include "factions\frwinter.sqf"
     };
+    
+    case "FRWOOD": {
+        #include "factions\frwinter.sqf"
```
Comment (Coryf88) [inline loadouts/faction-list.sqf]: Should be `#include "factions\frwood.sqf"`

File: loadouts/factions/structures/afrenian_army.sqf [hunk 2]
```diff
@@ -73,8 +73,7 @@ switch (true) do {
     case (_isTeamLeader): {
         [_unit, _leaderHEAD, _commonUNIFORM, _ftlVEST, "empty"] call BRM_FMK_fnc_useUniform;
         [_unit,[[_wsmoke,2],[_rsmoke,2],[_flashlight,1]]] call BRM_FMK_fnc_addtoVest;
-        [_unit, _commonRIFLEGL, _countRIFLE] call BRM_FMK_fnc_addWeaponKit;
```
Comment (Coryf88) [inline loadouts/factions/structures/afrenian_army.sqf]: This should be fixed in the [loadout](https://github.com/Neefay/BromA-Framework-A3-Addon/blob/master/loadouts/factions/afrenian.sqf#L28), not the structure.

File: loadouts/faction-list.sqf (hunk 1 above)
Comment (HeroesMaker) [reply loadouts/faction-list.sqf]: oh yeah nice, i haven't seen that thanks

File: loadouts/factions/structures/afrenian_army.sqf (hunk 2 above)
Comment (HeroesMaker) [reply loadouts/factions/structures/afrenian_army.sqf]: Afrenia army loadout use standalone GL, not rifle with GL, so the TL must have it's structure changed so he use a rifle, if you look at grenadier you'll see that he has both the rifle AND the standalone GL, also this structure is only for afrenia, not for everyone ( unless someone decide so )

**Pull request merged**